packages/cloud-cognitive/src/components/AddSelect/AddSelectBody.test.js

```diff
@@ -226,27 +268,45 @@ describe(componentName, () => {
   it('displays child items', () => {
     render(<AddSelectBody {...singleHierarchyProps} />);
     const childrenButton = document.querySelector(
-      `.${pkg.prefix}--add-select__selections-view-children`
+      `.${blockClass}__selections-view-children`
     );
     expect(screen.queryByText('Los Angeles')).toBeNull();
     fireEvent.click(childrenButton);
     expect(screen.queryByText('Los Angeles'));
   });
 
-  it('displays breadcrumbs', () => {
-    render(<AddSelectBody {...singleHierarchyProps} />);
-    const childrenButton = document.querySelector(
-      `.${pkg.prefix}--add-select__selections-view-children`
+  it('handles breadcrumbs', () => {
+    const normalizedItems = normalize(hierarchyItems);
+    const newProps = {
+      ...multiProps,
+      items: hierarchyItems,
+      useNormalizedItems: true,
+      normalizedItems,
+    };
+    render(<AddSelectBody {...newProps} />);
+    const childrenBtn = document.querySelectorAll(
+      `.${blockClass}__selections-view-children`
     );
     expect(document.querySelectorAll('.bx--breadcrumb-item').length).toEqual(1);
     expect(
       document.querySelectorAll('.bx--breadcrumb-item')[0].textContent
-    ).toBe('Categories');
-    fireEvent.click(childrenButton);
+    ).toBe('Business terms');
+    fireEvent.click(childrenBtn[0]);
     expect(document.querySelectorAll('.bx--breadcrumb-item').length).toEqual(2);
     expect(
       document.querySelectorAll('.bx--breadcrumb-item')[1].textContent
     ).toBe('California');
+    fireEvent.click(document.querySelectorAll('.bx--breadcrumb-item')[0]);
```

Comment:
We should probably be pulling in the Carbon prefix from:
```js
import { carbon } from '../../settings';

...
...
fireEvent.click(document.querySelectorAll(`.${carbon.prefix}--breadcrumb-item`)[0]);
...
...
```

Reply:
Same for the other instances of `.bx.....` within this file

Reply:
oh crap i forgot to replace the prefix's. thanks. will fix.